gen/definitions/high_availability.yaml

```diff
@@ -0,0 +1,145 @@
+---
+name: Device HA Pairs
```

Comment:
Can you rename the object to `Device HA Pair`. Plural form indicates that this is bulk resource and this is not.
Also, can you please rename the yaml file to match the resource/data source name, which would be fmc_device_ha_pair.yaml?